### PR TITLE
[Play] - Pregame Countdown

### PR DESCRIPTION
**Pull request by drewjhart**
This PR adds the pregame countdown page that occurs when the teacher moves from `GameSessionState.TEAMS_JOINING` to `GameSessionState.CHOOSE_CORRECT_ANSWER`

**Screenshot:**
Screenshot 2023-05-05 at 4 09 50 PM

**Figma Reference:**
image
[https://www.figma.com/file/PadzU8GqyhFLCjIt7rQkpE/RightOn-Apps?node-id=13554-169661&t=PGzsP6wda24aGPzP-4](**url**)

### CODE REVIEW

File: play/src/components/RadialTimer.tsx
Comment (maniramezan) [inline play/src/components/RadialTimer.tsx:1]: Might worth to have some comments or a link if this is inspired by any posts online

File: play/src/containers/GameSessionContainer.tsx [hunk 1]
```diff
@@ -14,7 +15,8 @@ import StartPhase2 from '../pages/StartPhase2';
 import { JoinGameState, FinalResultsState } from '../lib/PlayModels';
 
 export default function GameSessionContainer() {
-  const [gameSession, setGameSession] = useState(   // eslint-disable-line @typescript-eslint/no-unused-vars
+  const [gameSession, setGameSession] = useState(
+    // eslint-disable-line @typescript-eslint/no-unused-vars
```
Comment (maniramezan) [inline play/src/containers/GameSessionContainer.tsx:19]: If this is temporarily and needs to be clean up, one common practice is adding `// TODO: Remove this` or something with more info so you can easily find them and make sure these are resolved before final release

File: play/src/lib/Theme.tsx [hunk 2]
```diff
@@ -100,6 +102,7 @@ declare module '@mui/material/styles' {
     darkGrey?: string;
     lightGrey?: string;
     correctColor?: string;
+    countdownColor?: string;
```
Comment (maniramezan) [inline play/src/lib/Theme.tsx:105]: Any reason for this be nullable?

**Review by maniramezan — APPROVED** [review]

File: play/src/components/RadialTimer.tsx
Comment (drewjhart) [reply play/src/components/RadialTimer.tsx:1]: Thanks for the reviews on this PR and the others. Appreciate it as always.

And added reference comment back in (this is reused code from host/mobile)

File: play/src/containers/GameSessionContainer.tsx (hunk 1 above)
Comment (drewjhart) [reply play/src/containers/GameSessionContainer.tsx]: Added ``// TODO`` statements to explain this line and the lines below it

File: play/src/lib/Theme.tsx (hunk 2 above)
Comment (drewjhart) [reply play/src/lib/Theme.tsx]: Good point. Removed `?`

**Pull request merged**